index.html

```diff
@@ -238,7 +238,7 @@ <h3 class="feature-title">Develop</h3>
                                     </div>
                                     <h3 class="feature-title">Learn</h3>
                                     <p class="text-sm">Let's face it - learning code is just as difficult as learning a new language.
-                                    </br></br>We're continually adding multilingual tutorials to our repository that includes introductions to programming, #OpenSource how-to’s and the Legesher fundamentals!</p>
+                                    </br></br>We're continually adding multilingual tutorials to our repository that includes introduction to programming, #OpenSource how-to’s and the Legesher fundamentals!</p>
```

Comment:
Can you change the copy on this line to say something along the lines of "Stay tuned for multilingual tutorials for introduction to programming"? Would love to link our documentation website here, as that is the place that will host other tutorials and legesher documentation. 

Feel free to run your ideas by me!